joplin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -0,0 +1,42 @@
+{% load i18n %}
+{% if page_types_restricted %}
+    {% if page_type_string == "base.departmentpage" %}
```

Comment:
these conditionals are nested in a kinda messy way, but it does allow us to deduce what heading to have based on the context, which a fallback incase a picker gets used elsewhere